### PR TITLE
Added Docs for host configuration through environment variable

### PR DESCRIPTION
**Pull request by Colhodm**
Fixed the commits, so theres only one and docs are fixed

### CODE REVIEW

File: docs/minio-client-complete-guide.md [hunk 1]
```diff
@@ -142,6 +142,15 @@ mc config host add gcs  https://storage.googleapis.com BKIKJAA5BMMU2RHO6IBB V8f1
 
 NOTE: Google Cloud Storage only supports Legacy Signature Version 2, so you have to pick - S3v2
 
+### Specify AccessKey and SecretKey through environment variables
+```sh
+export MC_HOSTS_<alias> = https://<Access Key>:<Secret Key> @ <YOUR-S3-ENDPOINT>
```
Comment (kannappanr) [inline docs/minio-client-complete-guide.md]: There should not be any space around the `@` symbol

File: docs/minio-client-complete-guide.md [hunk 2]
```diff
@@ -142,6 +142,15 @@ mc config host add gcs  https://storage.googleapis.com BKIKJAA5BMMU2RHO6IBB V8f1
 
 NOTE: Google Cloud Storage only supports Legacy Signature Version 2, so you have to pick - S3v2
 
+### Specify AccessKey and SecretKey through environment variables
```
Comment (kannappanr) [inline docs/minio-client-complete-guide.md]: Can the title be changed to something else, as we are also specifying the endpoint here apart from the access credentials?

File: docs/minio-client-complete-guide.md [hunk 3]
```diff
@@ -142,6 +142,15 @@ mc config host add gcs  https://storage.googleapis.com BKIKJAA5BMMU2RHO6IBB V8f1
...
+```sh
+export MC_HOSTS_<alias> = https://<Access Key>:<Secret Key> @ <YOUR-S3-ENDPOINT>
+```
+Example:
+```sh
+export MC_HOSTS_myalias=https://Q3AM3UQ867SPQQA43P2F:zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG@play.minio.io:9000
+mc ls myalias //lists buckets
```
Comment (kannappanr) [inline docs/minio-client-complete-guide.md]: The comment here can be removed

File: docs/minio-client-complete-guide.md (hunk 1 above)
Comment (ebozduman) [reply docs/minio-client-complete-guide.md]: The same is true for the spaces around `=`. 
No space is allowed around `=` when exporting an environment variable.

**Pull request merged**